features/environment.py

```
@@ -0,0 +1,10 @@
+from allure_behave.hooks import allure_report
+
+
+from behave.runner import Context
+
+
+def before_all(context: Context) -> None:
```

Ruff (0.8.0)

Unused function argument: `context`

(ARG001)